src/Native/pkg/runtime.native.System.IO.Compression/runtime.native.System.IO.Compression.pkgproj

```diff
@@ -6,21 +6,19 @@
     <SkipValidatePackage>true</SkipValidatePackage>
   </PropertyGroup>
   <ItemGroup>
-    <!-- When updating ExternalExpectedPrerelease with a version that 
-         below should be updated to $(PackageVersion)-$(ExternalExpectedPrerelease) -->
     <RuntimeDependency Include="runtime.win10-x64-aot.runtime.native.System.IO.Compression">
       <TargetRuntime>win10-amd64-aot</TargetRuntime>
-      <Version>4.1.1-$(ExternalExpectedPrerelease)</Version>
+      <Version>$(RuntimeWin10RuntimeNativeSystemIOCompressionVersion)</Version>
```

Comment:
Done! Pushed an update adding the aot build to the pkgproj/builds and removed it as a runtime dependency.